src/Relation/Binary/Properties/Poset.agda

```diff
@@ -62,15 +63,10 @@ open Poset ≥-poset public
 ------------------------------------------------------------------------
 -- Negated order
 
-infix 4 _≰_
-
-_≰_ : Rel A p₃
-x ≰ y = ¬ (x ≤ y)
-
-≰-respˡ-≈ : _≰_ Respectsˡ _≈_
+≰-respˡ-≈ : _≰A_ Respectsˡ _≈_
```

Comment:
I'm getting a very weird rendering of the symbol here: `_≰A_`, in particular the `≰` symbol has an `A` after it? Weirdly it doesn't appear in `Bundles` though. Any ideas what is causing it?

Reply:
The `A` is there to avoid a renaming conflict; it's a locally introduced symbol at the top of the module, renaming apart from an existing symbol. See my comment in the `CHANGELOG` about "deprecation gymnastics"... or maybe there's a better way to proceed that you know?

Reply:
Ooh weird. Let me have a closer look at the renaming going on. Obviously didn't appreciate what was going on properly in the first review.

Reply:
BTW, something else weird is going on re: renaming/deprecation, on #2099: which I'll re-open temporarily to document.